x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/config_panel.test.tsx

```diff
@@ -126,7 +144,7 @@ describe('ConfigPanel', () => {
         first: mockDatasource.publicAPIMock,
         second: mockDatasource.publicAPIMock,
       };
-      const component = mountWithIntl(<LayerPanels {...defaultProps} />);
+      const component = mountWithIntl(<LayerPanels {...defaultProps} />, { attachTo: container });
```

Comment:
I found an alternate solution to this problem which just attached elements to `document.body`, any idea why that wouldn't be a simpler solution to this?

Reply:
Yes, you can do that, the test passes but then it prints for each instance:

```
 Console

    console.error
      Warning: render(): Rendering components directly into document.body is discouraged, since its children are often manipulated by third-party scripts and browser extensions. This may lead to subtle reconciliation issues. Try rendering into a container element created for your app.

      at warningWithoutStack (node_modules/react-dom/cjs/react-dom.development.js:530:32)
      at topLevelUpdateWarnings (node_modules/react-dom/cjs/react-dom.development.js:27449:114)
      at legacyRenderSubtreeIntoContainer (node_modules/react-dom/cjs/react-dom.development.js:27503:5)
      at Object.render (node_modules/react-dom/cjs/react-dom.development.js:27608:10)
      at fn (node_modules/enzyme-adapter-react-16/src/ReactSixteenAdapter.js:437:26)
      at node_modules/enzyme-adapter-react-16/src/ReactSixteenAdapter.js:354:37
      at batchedUpdates$1 (node_modules/react-dom/cjs/react-dom.development.js:24386:12)
      at Object.act (node_modules/react-dom/cjs/react-dom-test-utils.development.js:1092:14)
```

So I went for the silent solution 🔇